git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
Features or bug fixes that may span multiple commits might make this cumbersome.  I think our approach is to have the PR reference the JIRA ticket in the title "HYB-132 - Fix header bar offset" and also link in the description.  Do you think having every commit prefixed is needed?